common/ManagementTestShared/Redesign/ManagementRecordedTestBase.cs

```diff
@@ -84,6 +86,7 @@ protected ArmClient GetArmClient(ArmClientOptions clientOptions = default, strin
             options.Environment = GetEnvironment(TestEnvironment.ResourceManagerUrl);
             options.AddPolicy(ResourceGroupCleanupPolicy, HttpPipelinePosition.PerCall);
             options.AddPolicy(ManagementGroupCleanupPolicy, HttpPipelinePosition.PerCall);
+            options.AddPolicy(NullFilterPolicy, HttpPipelinePosition.PerCall);
```

Comment:
Should we add this policy into the PerRetry policy pipeline? I assume this policy needs execution every time we get a response